File: test/fixtures/output/java/restclient/application-form-encoded.java
RestClient restClient = RestClient.create();

MultiValueMap<String, String> formDataMap = new LinkedMultiValueMap<>();
formDataMap.add("foo", "bar");
formDataMap.add("hello", "world");

ResponseEntity<String> response = restClient
  .method(HttpMethod.POST)
  .uri("http://mockbin.com/har")
  .contentType(MediaType.APPLICATION_FORM_URLENCODED)
  .body(formDataMap)
  .retrieve()
  .toEntity(String.class);

File: test/fixtures/output/csharp/restsharp/application-form-encoded.cs
var client = new RestClient("http://mockbin.com");
var request = new RestRequest("/har", Method.Post);
request.AddHeader("content-type", "application/x-www-form-urlencoded");
request.AddParameter("application/x-www-form-urlencoded", "foo=bar&hello=world", ParameterType.RequestBody);
var response = client.Execute(request);